test/benchmark/shell_scripts/iGenVar_statistics_via_truvari.sh

```diff
@@ -0,0 +1,100 @@
+
+exec 3>&1 4>&2 # store original streams
+trap 'exec 2>&4 1>&3' 0 1 2 3 # restore original streams when scripts ends
+exec 1>logs/${current_date}_statistics_via_truvari.log 2>&1 # write stdout and stdcerr to logfile
+
+set -ex
+
+echo $(date)
+start=$(date +%s) # get starting date
+
+# -------- -------- pre installation steps -------- -------- #
+# for truvari you need python3,
+# we recommend to use miniconda:
+# ------------------------------
+# wget https://repo.anaconda.com/miniconda/Miniconda2-latest-Linux-x86_64.sh
+# chmod +x Miniconda2-latest-Linux-x86_64.sh
+# sh Miniconda2-latest-Linux-x86_64.sh
+
+# conda env create -f environment.yml
+
+# source /home/USER_DIR_PATH/.bashrc                                    # if the .bashrc is not executed automatically
+conda activate iGenVar_benchmark
+
+# -------- -------- get truth set ../../data -------- -------- #
+cd ../../data
+mkdir -p truth_set && cd truth_set
+
+wget --retry-connrefused --waitretry=30 --read-timeout=30 --timeout=30 --tries=20 --no-clobber --no-verbose \
+    https://ftp-trace.ncbi.nlm.nih.gov/ReferenceSamples/giab/../../data/AshkenazimTrio/analysis/NIST_SVs_Integration_v0.6/HG002_SVs_Tier1_v0.6.vcf.gz
+wget --retry-connrefused --waitretry=30 --read-timeout=30 --timeout=30 --tries=20 --no-clobber --no-verbose \
+    https://ftp-trace.ncbi.nlm.nih.gov/ReferenceSamples/giab/../../data/AshkenazimTrio/analysis/NIST_SVs_Integration_v0.6/HG002_SVs_Tier1_v0.6.vcf.gz.tbi
+wget --retry-connrefused --waitretry=30 --read-timeout=30 --timeout=30 --tries=20 --no-clobber --no-verbose \
+    https://ftp-trace.ncbi.nlm.nih.gov/ReferenceSamples/giab/../../data/AshkenazimTrio/analysis/NIST_SVs_Integration_v0.6/HG002_SVs_Tier1_v0.6.bed
+
+echo "$(tput setaf 1)$(tput setab 7)------- truth set downloaded (1/3) --------$(tput sgr 0)" 1>&3
+
+cd ../..
+
+
+# -------- -------- init for truvari -------- -------- #
+
+cd results/${current_date}
+
+for min_qual in 1 5 10
+do
+    echo "Looping min_qual number $min_qual"
+
+    for hierarchical_clustering_cutoff in 10 50 100 150
+    do
+        echo "Looping hierarchical_clustering_cutoff number $hierarchical_clustering_cutoff"
+
+        vcf_file_name=${min_qual}_${hierarchical_clustering_cutoff}_output.vcf
+        bgzip -c ${vcf_file_name} > ${vcf_file_name}.gz
+        tabix -p vcf ${vcf_file_name}.gz
+    done
+done
+
+echo "$(tput setaf 1)$(tput setab 7)------- truvari init done (2/3) --------$(tput sgr 0)" 1>&3
+
+# -------- -------- run truvari -------- -------- #
+
+for min_qual in 1 5 10
+do
+    echo "Looping min_qual number $min_qual"
+
+    for hierarchical_clustering_cutoff in 10 50 100 150
+    do
+        echo "Looping hierarchical_clustering_cutoff number $hierarchical_clustering_cutoff"
+        vcf_file_name=${min_qual}_${hierarchical_clustering_cutoff}_output.vcf
+
+        # find ./ -type d -name "${min_qual}_${hierarchical_clustering_cutoff}_truvari_default" -exec rm -rf {} +        # delete if already exists
```

Comment:
I'm not sure whether you want to delete it or not. The `find` command will delete `"${min_qual}_${hierarchical_clustering_cutoff}_truvari_default"` if it exists and otherwise do nothing. There won't be an error if the directory did not exist (in contrast to when calling `rm`).

Reply:
Truvari makes a new directory and gives an error if it exists, thats why I wanted to delete it before. But now I changed the directory name to include a date, so I may want to get that error. I'll have to discuss this with @eldariont. Maybe we will delete this part anyway, since we are thinking of using snake make.